lightfm/lightfm.py

```diff
@@ -94,6 +94,10 @@ class LightFM(object):
     random_state: int seed, RandomState instance, or None
         The seed of the pseudo random number generator to use when shuffling
         the data and initializing the parameters.
+    normalize: bool, optional
```

Comment:
Can we reformulate this to better reflect the fact that this affects training as well? Something like the following:

If True, user and item vectors will be rescaled to unit length during training and prediction.